librz/bin/format/elf/elf.c

```diff
@@ -1415,6 +1415,143 @@ ut64 Elf_(rz_bin_elf_get_section_addr_end)(ELFOBJ *bin, const char *section_name
 	RzBinElfSection *section = get_section_by_name(bin, section_name);
 	return section ? section->rva + section->size : UT64_MAX;
 }
+char* Elf_(rz_bin_elf_get_section_flag)(struct rz_bin_elf_section_t * section) {
+	char *buff;
+	//RzList *buff = rz_list_new();//rz__str_new("")
+	//rz_str_append(buff, rz_string_new("Write"));
+	int8_t i=0;
+	buff = (char*)malloc(2*sizeof(char));
```

Comment:
buff is uninitialized

Reply:
This function is not being used anymore. This code will be removed. The implementation of getting the section flags is under bin_elf.inc right now.